### ISSUE TITLE
HTML in Markdown rendered incorrectly 

### ISSUE THREAD

**Issue by okonet** [styleguidist/react-styleguidist#402]
This snippet in the markdown document:

```md
<a href="https://opencollective.com/react-dropzone/sponsor/0/website" target="_blank"><img src="https://opencollective.com/react-dropzone/sponsor/0/avatar.svg"></a>
```

is rendered to:

![2017-04-20 at 13 43](https://cloud.githubusercontent.com/assets/11071/25229117/52b63c96-25cf-11e7-992d-27ea1f5ac8d5.png)

which isn't correct.

To reproduce, create a styleguide config with 

```
sections: [
    {
      content: 'README.md'
    }
]
```

and update README to have such snippet.

**Comment by sapegin** [comment]
I think you should report it to [markdown-to-jsx](https://github.com/probablyup/markdown-to-jsx).
